components/sample-data-service/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../../Makefile.common_go
+
+PACKAGE_PATH = github.com/chef/automate/components/sample-data-service
+
+BINS = ${PACKAGE_PATH}/cmd/sample-data-service
+
+packages:=${PACKAGE_PATH}/...
+ifdef CI
+  verbose?="-v"
+endif
+
+all: lint build test
```

Comment:
The `test` target is missing? Are there tests to run? If not, remove it here.

Reply:
Right now, have added these helpers to the future building, removing test make sense we can add these helpers once required. Thanks!